src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/dataset_retriever.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
qq: is this the same license for other files in the repo? I think a lot of files have different format of license (I don't know if the two are same or different)

https://github.com/triton-inference-server/client/blob/main/src/c%2B%2B/perf_analyzer/genai-perf/genai_perf/main.py